terraform/gpu/variables.tf

```diff
@@ -0,0 +1,58 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+variable "test_dir" {
+  type    = string
+  default = "../../test/gpu"
+}
+
+variable "addon_name" {
+  type    = string
+  default = "amazon-cloudwatch-observability"
+}
+
+variable "addon_version" {
+  type    = string
+  default = "latest"
+}
+
+
+variable "cwagent_image_repo" {
```

Comment:
Is this used?

Reply:
ahh, forgot to remove that, was using it for testing. Removed anything unused from terraorm